exec/java-exec/src/main/resources/rest/storage/list.ftl

```diff
@@ -92,10 +92,10 @@
                 <button type="button" class="btn btn-primary" onclick="location.href='/storage/${plugin.getName()}'">
                   Update
                 </button>
-                <button type="button" class="btn btn-primary" onclick="doEnable('${plugin.getName()}', true)">
+                <button type="button" class="btn btn-success" onclick="doEnable('${plugin.getName()}', true)">
                   Enable
                 </button>
-                <button type="button" class="btn btn-primary" name="${plugin.getName()}" data-toggle="modal"
+                <button type="button" class="btn!" name="${plugin.getName()}" data-toggle="modal"
```

Comment:
`btn!` seems to be a typo. 
Results in the Export buttons for Disabled plugins to not be rendered in a standardized way.
![image](https://user-images.githubusercontent.com/4335237/54705617-7531fb00-4afa-11e9-88db-9755cf4472d8.png)